api/kms/kms.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package sidero.kms;
+
+option go_package = "github.com/siderolabs/kms/api/kms";
+
+// KMSService provides APIs to encrypt and decrypt data.
+service KMSService {
+  rpc Seal(Request) returns (Response);
+  rpc Unseal(Request) returns (Response);
+}
+
+message Request {
+  // Node UUID as string.
+  string node_uuid = 1;
+  // Data to encrypt or decrypt.
+  bytes data = 2;
```

Comment:
we should probably limit the `data` length here, more for consistency reasons? 

Seal/Unseal have a bit different requirements, but still

Reply:
What limit do you think will make sense?

Reply:
this is another good topic, I was going to ask this question in Talos PR. 

e.g. can we do 32-bytes random encryption key, that should be more than enough.

so on Seal, we can say `len(data) == 32`, and Unseal should be `32 + 12` always

This should match TPM key size

Reply:
Looks like TPM key size is in range 32 - 128 byte

Reply:
the actual encryption key (unsealed one) is 32 bytes: https://github.com/systemd/systemd/blob/06afda6b38d5d730fca3c65449096425933272bc/src/shared/tpm2-util.c#L2918-L2937

Reply:
> and Unseal should be 32 + 12 always

I don't think it's true: encrypted data is 48 bytes + nonce it gets to 60.

Reply:
that's because AES has fixed block of 16 bytes. So I guess I can just make it 60

Reply:
yep, sorry, sealed is longer due to auth, thanks

Reply:
fixed